tools/includes/version-compare.sh

```diff
@@ -88,3 +88,88 @@ function version_compare {
 		[[ ${#A[@]} -gt ${#B[@]} ]]
 	fi
 }
+
+# Compute the difference between two version numbers, semver style.
+#
+# Note this is a bit looser than `pnpm semver`, as it accepts 4-part versions.
+#
+# @param $1 First version.
+# @param $2 Second version.
+# Outputs: equal, suffix, patch, minor, or major.
+function version_diff {
+	# Variable for the version_difference call
+	local INCREMENT
+
+	# Returning 0 if neither $1 > $2, nor $2 > $1
+	if ! version_compare "$1" "$2" 1 && ! version_compare "$2" "$1" 1; then
+		echo "equal"
```

Comment:
Need to end here or it'll output more later on.

Reply:
Yep, added a `return`, thanks!